File: Pythonleetcode/Leetcode1662.py
class Solution(object):
    def arrayStringsAreEqual(self, word1, word2):
        string1=""
        string2=""
        for i in word1:
            string1+=i
        for i in word2:
            string2 += i
        return string1==string2

print(Solution().arrayStringsAreEqual(word1 = ["ab", "c"], word2 = ["a", "bc"]))
print(Solution().arrayStringsAreEqual(word1 = ["a", "cb"], word2 = ["ab", "c"]))
print(Solution().arrayStringsAreEqual(word1  = ["abc", "d", "defg"], word2 = ["abcddefg"]))

File: CppPractice/Project1/Project1/Leetcode1662.cpp
#include <string>

#include <vector>

#include <iostream>

#include <stdlib.h>

#include <time.h>

#include <sstream>

#include <algorithm>

#include <map>

using namespace std;


class Solution {
public:
    bool arrayStringsAreEqual(vector<string>& word1, vector<string>& word2){
        string ans1;
        string ans2;
        for (int i = 0; i < word1.size(); i++)
        {
            ans1 += word1[i];
        }
        for (int i = 0; i < word2.size(); i++)
        {
            ans2 += word2[i];
        }
        return ans1 == ans2;
    }
};